addons/l10n_nl/models/account_chart_template.py

```diff
@@ -9,5 +9,9 @@ class AccountChartTemplate(models.AbstractModel):
     def _post_load_data(self, template_code, company, template_data):
         super()._post_load_data(template_code, company, template_data)
         if template_code == 'nl':
-            company.account_journal_suspense_account_id.tag_ids += self.env.ref('l10n_nl.account_tag_25')
-            company.get_unaffected_earnings_account().tag_ids += self.env.ref('l10n_nl.account_tag_12')
+            cash_tag = self.env.ref('l10n_nl.account_tag_25', raise_if_not_found=False)
```

Comment:
Does it really work to do the `+=` if it is not found?

Reply:
In what case would it be not found? Would not the tag be always there at CoA loading time?